### PR TITLE
zh-translation:/zh/docs/ops/common-problems/network-issues/index.md

### PR DESCRIPTION
**Pull request by dk-lockdown**
ref: https://github.com/servicemesher/istio-official-translation/issues/1237

### CODE REVIEW

**Comment by rootsongjc** [comment]
Request for a reviewer.

File: content/zh/docs/ops/common-problems/network-issues/index.md [hunk 1]
```diff
@@ -342,30 +296,22 @@ server {
...
+## 当为多个 gateway 配置了相同的 TLS 证书导致 404 异常{#404-errors-occur-when-multiple-gateways-configured-with-same-tls-certificate}
 
-Configuring more than one gateway using the same TLS certificate will cause browsers
-that leverage [HTTP/2 connection reuse](https://httpwg.org/specs/rfc7540.html#reuse)
-(i.e., most browsers) to produce 404 errors when accessing a second host after a
-connection to another host has already been established.
+使用相同的 TLS 证书配置到超过一个 gateway，当浏览器与另一个主机建立了连接访问第二个主机时会导致浏览器快取 [HTTP/2 连接复用](https://httpwg.org/specs/rfc7540.html#reuse)（例如，大部分浏览器）从而导致 404 异常产生。
 
-For example, let's say you have 2 hosts that share the same TLS certificate like this:
+举个例子，假如你有2个主机共用相同的TLS证书，如下所示：
```
Comment (vflong) [inline content/zh/docs/ops/common-problems/network-issues/index.md]: - 中文和英文之间加空格
- 中文与数字之间、英文与数字之间都需要加空格

File: content/zh/docs/ops/common-problems/network-issues/index.md [hunk 2]
```diff
@@ -1,81 +1,57 @@
...
-Common response flags are:
+通用响应标志如下：
 
-- `NR`: No route configured, check your `DestinationRule` or `VirtualService`.
-- `UO`: Upstream overflow with circuit breaking, check your circuit breaker configuration in `DestinationRule`.
-- `UF`: Failed to connect to upstream, if you're using Istio authentication, check for a
-[mutual TLS configuration conflict](#503-errors-after-setting-destination-rule).
+- `NR`: 没有配置路由，请检查你的 `DestinationRule` 或者 `VirtualService` 配置。
+- `UO`: 上游溢出导致断路，请在 `DestinationRule` 检查你的熔断器配置。
+- `UF`: 未能连接到上游，如果你正在使用 Istio 认证，请检查[mutual-tls 配置冲突](#503-errors-after-setting-destination-rule)。
```
[the remark this answers is not in the document]
Comment (dk-lockdown) [reply content/zh/docs/ops/common-problems/network-issues/index.md]: #503-errors-after-setting-destination-rule，这个锚点会导致lint错误，这个怎么解决啊

[the remark this answers is not in the document]
Comment (vflong) [reply content/zh/docs/ops/common-problems/network-issues/index.md]: (#`503-errors-after-setting-destination-rule`)
---
这样试试？

**Pull request closed without merging**